bigquery_etl/cli/utils.py

```diff
@@ -120,6 +128,56 @@ def paths_matching_name_pattern(pattern, sql_path, project_id, files=["*.sql"]):
     return matching_files
 
 
+def qualified_table_name_matching(qualified_table_name) -> Tuple[str, str, str]:
+    """Match qualified table name pattern."""
+    if match := QUALIFIED_TABLE_NAME_RE.match(qualified_table_name):
+        project_id = match.group("project_id")
+        dataset_id = match.group("dataset_id")
+        table_id = match.group("table_id")
+    else:
+        raise AttributeError(
+            "Qualified table name must be named like:" + " <project>.<dataset>.<table>"
+        )
+
+    return (project_id, dataset_id, table_id)
+
+
+def backfills_matching_name_pattern(
+    sql_dir, project_id, qualified_table_name
+) -> defaultdict[Path, List[Backfill]]:
+    """Return backfills with name pattern."""
```

Comment:
nit: could you say what the keys of the returned dict are supposed to represent here (or in the function name?) I assume it's the path to the backfill file? Overall this function seems more like a backfill util than a CLI util

Reply:
Yes the keys represent the path to the backfill file.